Corona-Warn-App/src/main/java/de/rki/coronawarnapp/installTime/InstallTimeProvider.kt

```diff
@@ -0,0 +1,18 @@
+package de.rki.coronawarnapp.installTime
+
+import android.content.Context
+import de.rki.coronawarnapp.util.di.AppContext
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class InstallTimeProvider @Inject constructor(
+    @AppContext private val context: Context
+) {
+    private val installTime: Long = context
+        .packageManager
+        .getPackageInfo(context.packageName, 0)
+        .firstInstallTime
+
+    val daysSinceInstallation: Long get() = System.currentTimeMillis() - installTime
```

Comment:
is it days or milliseconds since installation?

Reply:
It's millis, good point. I'll change it.